app/controllers/classifier.coffee

```diff
@@ -90,6 +93,8 @@ class Classifier extends Controller
     @subjectViewer.setClassification @classification
     @animalSelector.setClassification @classification
 
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,subject.zooniverseId,'view',''
```

Comment:
Selecting a subject happens automatically on load. The user doesn't see it until they visit the Classify page.